python/_brotli.cc

```diff
@@ -88,6 +87,37 @@ static int lgblock_convertor(PyObject *o, int *lgblock) {
   return 1;
 }
 
+static BROTLI_BOOL compress_stream(BrotliEncoderState* enc, BrotliEncoderOperation op,
+                                   std::vector<uint8_t>* output, uint8_t* input, size_t input_length) {
+  BROTLI_BOOL ok = BROTLI_TRUE;
+
+  while (ok) {
+    size_t available_in = input_length;
+    const uint8_t* next_in = input;
+    size_t available_out = 0;
+
+    ok = BrotliEncoderCompressStream(enc, op,
+                                     &available_in, &next_in,
+                                     &available_out, NULL, NULL);
```

Comment:
Found the problem: "take output" does not transition out from "flush requested" state.
Going to fix ASAP.